maven-scm-providers/maven-scm-providers-git/maven-scm-provider-jgit/src/test/java/org/apache/maven/scm/provider/git/jgit/command/checkin/JGitCheckInCommandCommitterAuthorTckTest.java

```diff
@@ -63,7 +69,7 @@ protected void setUp()
     }
     
     @Override
-    protected void tearDown()
+    public void tearDown()
```

Comment:
Apparently, this still works (at least, no tests seem to be failing because this doesn't run) but wouldn't it be more nice to annotate with `@After` (in JUnit 4 style)?

Reply:
fixed